recipes/bokeh/run_test.py

```diff
@@ -0,0 +1,8 @@
+import sys
+import bokeh
+
+if sys.platform != 'win32':
+    bokeh.test()
+
+print('bokeh.__version__: %s' % bokeh.__version__)
+assert bokeh.__version__ == '0.12.0'
```

Comment:
Version here needs to be 0.11.1.  Could also make it be os.environ['PKG_VERSION'] I think - but I don't totally remember if this variable is still defined at test time.